index.html

```diff
@@ -46,6 +46,9 @@ <h2>Indianapolis, IN</h2>
         <li><a href="/youtube">Watch past talks on YouTube</a></li>
         <li><a href="/discord">Join our Discord</a></li>
         <li><a href="/getting-started">WebDev "Getting Started" Resources</a></li>
+          <ul>
+            <li><a href="/hacktoberfest">Hacktoberfest</a></li>
+          </ul>
```

Comment:
Sub-lists should be nested in a list item
```suggestion
        <li>
          <a href="/getting-started">WebDev "Getting Started" Resources</a>
          <ul>
            <li><a href="/hacktoberfest">Hacktoberfest</a></li>
          </ul>
        </li>
```